### PR TITLE
Check if a service has already been restarted (Fixes #1338)

### LINKED ISSUE #1338: Add a check if a service have been already restarted
For instance on ArchLinux `/lib` is symlinked to `/usr/lib` thus all the services will be found twice (in both `/usr/lib` and `/lib`)

### PR DESCRIPTION
**Pull request by codeswhite**

### CODE REVIEW

File: updateHostsFile.py [hunk 1]
```diff
@@ -1312,6 +1316,7 @@ def flush_dns_cache():
                         print_failure(service_msg.format(result="failed"))
                     else:
                         print_success(service_msg.format(result="succeeded"))
+                    restarted_services += [service]
```
Comment (funilrys) [inline updateHostsFile.py:1319]: Please prefer `restarted_services.append()` as used in the rest of the script.

It's not for us, it for future Python contributors.

Comment (codeswhite) [reply updateHostsFile.py]: Understandable, done.

**Comment by StevenBlack** [comment]
Thanks @codeswhite and Nissar @funilrys.

Merging!

**Comment by StevenBlack** [comment]
@codeswhite and Nissar @funilrys, along with @XhmikosR, the great work you've done recently on code prompts me to make the latest a "minor" release, as opposed to a "patch" release.

Therefore we are now at [Release 2.70](https://github.com/StevenBlack/hosts/releases/tag/2.7.0) from [Release 2.6.29](https://github.com/StevenBlack/hosts/releases).

THANK YOU!

**Pull request merged**